test/hotspot/jtreg/compiler/vectorization/TestFloat16VectorSum.java

```diff
@@ -42,7 +43,7 @@ public class TestFloat16VectorSum {
     private Random rng;
 
     public static void main(String args[]) {
-        TestFramework.run(TestFloat16VectorSum.class);
+        TestFramework.runWithFlags("--enable-preview", "-XX:-TieredCompilation", "-Xbatch");
```

Comment:
Hi Jatin, thanks for this patch. May I ask, why `-XX:-TieredCompilation` option is passed? It's one of the whitelisted flags and must be already applied when compiling the test isn't it?

Reply:
Does this test works for you ?
I faced some failure with this test despite of passing --enable-preview, this works in the standalone (non-jtreg) mode though.  Somehow --enable-preview option is not being passed while loading one of the test harness class (OutputBuffer) which has preview specific minor version. This could be gap in IR framework, but I need to confirm it.

On your query, white listing filters out passing any non-whitelisted option to IR framework.

Reply:
Yes, I also get the same error as you. Somehow the test passes when the `@enablePreview` flag is removed from line #28. I am not sure why though. I can see FP16 vectorized code on aarch64 as well. Thanks.

Reply:
For now @enablePreview can be removed, since test codes does not contain any feature necessitating  --enable-preview during compilation.